service/src/test/java/bio/terra/workspace/service/resource/controlled/ControlledResourceServiceTest.java

```diff
@@ -1269,7 +1282,8 @@ void updateBqDatasetWithUndefinedExpirationTimes() throws Exception {
         new ApiGcpBigQueryDatasetUpdateParameters()
             .defaultTableLifetime(0L)
             .defaultPartitionLifetime(0L);
-    controlledResourceService.updateBqDataset(resource, updateParameters, null, null);
+    controlledResourceService.updateBqDataset(
+        resource, updateParameters, null, null, userAccessUtils.defaultUserAuthRequest());
```

Comment:
nit: maybe could have /* */ for the 2 null parameters.  but not really necessary for the test, maybe